src/views/Phrases.vue

```diff
@@ -47,31 +47,41 @@ export default {
           this.$forceUpdate();
           // tell the user that contiguous selections are required
           alert(msg);
-          console.log("done && tokenObj.isSelected....", "start", start, "end",end);
+          console.log(
+            "done && tokenObj.isSelected....",
+            "start",
+            start,
+            "end",
+            end
+          );
```

Comment:
Before merging, we should remove all console.log statements that aren't explicitly used in production.

Reply:
Sorry forgot to do that!